skyportal/models/earthquake.py

```diff
@@ -0,0 +1,177 @@
+__all__ = ['EarthquakeEvent', 'EarthquakeNotice', 'EarthquakePrediction']
+
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm import deferred
+
+from baselayer.app.models import (
+    Base,
+    AccessibleIfUserMatches,
+)
+
+from baselayer.app.env import load_env
+
+_, cfg = load_env()
+
+
+class EarthquakeNotice(Base):
+    """Earthquake notice information"""
+
+    update = delete = AccessibleIfUserMatches('sent_by')
+
+    sent_by_id = sa.Column(
+        sa.ForeignKey('users.id', ondelete='CASCADE'),
+        nullable=False,
+        index=True,
+        doc="The ID of the User who created this GcnEvent.",
+    )
+
+    sent_by = relationship(
+        "User",
+        foreign_keys=sent_by_id,
+        back_populates="earthquakenotices",
+        doc="The user that saved this GcnEvent",
+    )
+
+    content = deferred(
+        sa.Column(sa.LargeBinary, nullable=True, doc='Raw QuakeML content')
+    )
+
+    event_id = sa.Column(
+        sa.String,
+        sa.ForeignKey("earthquakeevents.event_id"),
+        nullable=False,
+        comment='Earthquake ID',
+    )
+
+    lat = sa.Column(sa.Float, nullable=False, comment='Latitude')
+
+    lon = sa.Column(sa.Float, nullable=False, comment='Longitude')
+
+    depth = sa.Column(sa.Float, nullable=False, comment='Depth')
```

Comment:
Is this relative to sea level? I'm considering if negative elevation is better here. But definitely want to define depth more carefully.